app/components/OrganizationMemberList.tsx

```diff
@@ -0,0 +1,161 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import Dropdown from '../icons/TriangleDown'
+import { useState } from 'react'
+import { Checkbox } from './Checkbox'
+import { PageIndicator } from './PageIndicator'
+import { ProfilePicture } from './ProfilePicture'
+
+export enum Role {
+  user,
+  admin,
+}
+
+type OrganizationMemberListTranslation = {
+  edit: string,
+  remove: string,
+  deselectAll: string,
+  selectAll: string,
+  members: string,
+  addMember: string,
+  saveChanges: string,
+  role: string,
+  of: string,
+  roleTypes: (role: Role) => string
```

Comment:
Functions are a nice thing to have in the translation objects but we shouldn't overuse them, they are pretty much just meant for things where you would otherwise use some kind of format string and insert values into it

In this case the following would be better:
```ts
export type OrganizationMemberListTranslation = {
  edit: string,
  // ...
  roles: Record<Role, string>
}
```